pkg/api/install/install_test.go

```diff
@@ -57,7 +57,7 @@ func TestCodec(t *testing.T) {
 	if err := json.Unmarshal(data, &other); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	if other.APIVersion != latest.GroupOrDie("").Version || other.Kind != "Pod" {
+	if other.APIVersion != latest.GroupOrDie("").GroupVersion.Version || other.Kind != "Pod" {
```

Comment:
> 1. Each group/version has a SchemeGroupVersion. To use it, one needs to call e.g., v1.SchemeGroupVersion.Version, which hardcodes the version in the package.

Again, I'm not going to make it an issue for this pull, but having a hardcoded package dependency upon the package that you depend on is an entirely reasonable thing to do and allows for easy analysis of dependencies.